cranelift/codegen/src/isa/x64/abi.rs

```diff
@@ -957,25 +957,39 @@ fn adjust_stack<C: LowerCtx<I = Inst>>(ctx: &mut C, amount: u64, is_sub: bool) {
 }
 
 fn load_stack(mem: impl Into<SyntheticAmode>, into_reg: Writable<Reg>, ty: Type) -> Inst {
-    let ext_mode = match ty {
-        types::B1 | types::B8 | types::I8 => Some(ExtMode::BQ),
-        types::B16 | types::I16 => Some(ExtMode::WQ),
-        types::B32 | types::I32 => Some(ExtMode::LQ),
-        types::B64 | types::I64 => None,
-        types::F32 => todo!("f32 load_stack"),
-        types::F64 => todo!("f64 load_stack"),
-        _ => unimplemented!("load_stack({})", ty),
+    let (is_int, ext_mode) = match ty {
```

Comment:
Oh, maybe that's a stupid idea, because F64 and F32s live in V128 class registers, and we need a way to distinguish between these three cases.  So ignore this.